src/test/java/org/training/poker/AppTest.java

```diff
@@ -1,20 +1,23 @@
 package org.training.poker;
 
+import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
 import org.junit.Test;
 
 /**
  * Unit test for simple App.
  */
-public class AppTest 
-{
-    /**
-     * Rigorous Test :-)
-     */
-    @Test
-    public void shouldAnswerWithTrue()
-    {
-        assertTrue( true );
-    }
+public class AppTest {
```

Comment:
Can you change the class name to PokerTest ?
```suggestion
public class PokerTest {
```